frontend/app/src/i18n/en.json

```diff
@@ -363,7 +363,9 @@
+        "referral_rewards_a": "As a Diamond member you can earn rewards for each user you refer who goes on to become a Diamond member themselves. You will receive half of any Diamond payments they make within the first year of joining OpenChat. You can find your referral link in the \"invite friends/family\" section of your \"profile settings\" from the main menu.<br><br>How might you do this? There are lots of ways. You can tell friends and family, tweet about OpenChat, create a video about OpenChat describing what you love about it and share it to TikTok or YouTube. You could stream on Twitch or write a blog post or create a podcast or host a Twitter space or even hold a real life meet-up. We look forward to seeing all the creative ideas that you come up with to get the word out!<br><br>In the future we'll open up referral rewards for all users you refer regardless of whether they become Diamond members. However because this is easily exploitable we'll first need to introduce some type of \"proof of unique personhood\".",
+        "translation_q": "How can I correct a translation error?",
+        "translation_a": "Our translations are provided by Google translate by default and sometimes there is just not enough context for Google to do a good job. It often takes a native language speaker to provide a better translation. If you have selected one of our non-english supported languages in your profile, you will notice that there is also a toggle in your profile to activate translation correction mode. With this mode enabled, you will see a small translate icon next to each label within the UI that has been translated. If you click this icon you will then be able to submit a suggested correction. Try to come up with a suggestion that is both accurate and fits well into the UI. You can check how your suggestion looks in place and repeat the procedure as many times as you like. Your suggestion will be reviewed by the team and will be adopted if approved. We really appreciate you taking the time to make our translations better - we couldn't do it without you."
```

Comment:
Probably worth also mentioning here they will be paid in CHAT for each approved correction